common/annotations/de_CH.xml

```diff
@@ -12,324 +12,28 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<territory type="CH"/>
 	</identity>
 	<annotations>
-		<annotation cp="{">↑↑↑</annotation>
```

Comment:
It looks like all of the ↑↑↑ lines are disappearing. That's ok; I'm a little surprised that they were in the last release.

Reply:
It was a separate bug that was fixed.